omod/src/main/java/org/openmrs/module/pihmalawi/RestUtils.java

```diff
@@ -38,4 +40,13 @@ public static ResponseEntity<Map<String, Object>> errorResponse(Throwable t) {
         data.put("errorMessages",errorMessages);
         return ResponseEntity.status(HttpStatus.INTERNAL_SERVER_ERROR).body(data);
     }
+
+    public static boolean isMacAddressAllowed(String macAddress) {
+        boolean allowed = false;
+        if (StringUtils.isNotBlank(macAddress)) {
+            String addresses = Context.getAdministrationService().getGlobalProperty(PihMalawiConstants.SYNC_ALLOW_MAC_ADDRESSES_GP_NAME);
```

Comment:
Pro tip:  Use the core `ConfigUtil` class to easily get a property.  This leverages caching (by implementing GlobalPropertyListener and maintaining a cache), as well as having a method to simply `getProperty(String property)`, which allows one to define the property via either a System Property, Runtime Property, or Global Property, with inheritance in that order...